vep/__init__.py

```diff
@@ -25,53 +30,68 @@ def add_cli_arguments(self, parser):
 
         group.add_argument(
             '--package-prefix',
-            default = '/usr/local',
-            help = "Path to prefix the entire package with"
+            default='/usr/local',
+            help="Path to prefix the entire package with"
         )
 
         group.add_argument(
             '--repo-url',
-            default = pycmd('setup.py', '--url').strip(),
-            help = "Repo URL to pass through to fpm"
+            default=pycmd('setup.py', '--url').strip(),
+            help="Repo URL to pass through to fpm"
         )
 
         group.add_argument(
             '--package-name',
-            default = pycmd('setup.py', '--name').strip(),
-            help = "The package name, as seen in apt"
+            default=pycmd('setup.py', '--name').strip(),
+            help="The package name, as seen in apt"
         )
 
         group.add_argument(
             '--package-version',
-            default = pycmd('setup.py', '--version').strip(),
-            help = "The package version."
+            default=pycmd('setup.py', '--version').strip(),
+            help="The package version."
+        )
+
+        group.add_argument(
+            '--skip-scripts',
+            default=False,
+            action='store_true',
+            help="Set this to skip installing all the scripts in all the setup.py files in all the requirements"
+        )
+
+        group.add_argument(
+            '--shim-script',
+            default=None,
+            help="An extra script to run between the build and package steps. "
+                 "If you need to do unnatural things to make your package work, this is the place to do them. "
+                 "Called scripts need to have a shebang line."
```

Comment:
This should probably be a little clearer. I assume from above comments that the scripts are executed with `sh`, so maybe just say that in the help?